assets/css/style.css

```diff
@@ -420,22 +421,22 @@ input[type="text"] {
 
 #deadline {
   color: #757575;
-  font-family: "Darker Grotesque", sans-serif;
+  font-family: 'Darker Grotesque', sans-serif;
   font-size: 1.6rem;
   width: 60%;
 }
 
-input[type="date"]::-webkit-inner-spin-button,
-input[type="date"]::-webkit-calendar-picker-indicator {
+input[type='date']::-webkit-inner-spin-button,
+input[type='date']::-webkit-calendar-picker-indicator {
   -webkit-appearance: none;
   display: none;
 }
 
 /* Error Messages */
 
 .errorFonts {
-  color: #F8380E;
-  font-family: "Darker Grotesque", sans-serif;
+  color: #f8380e;
```

Comment:
I prefer uppercase hex codes but I guess it's just preference

Reply:
I didn't change it. It's prettier's fault again lol